src/Radio/index.js

```diff
@@ -49,12 +49,12 @@ class Radio extends Component {
         super(props)
 
         this.state = {
-            focus: !!props.initialFocus,
+            focus: props.initialFocus,
```

Comment:
`initialFocus` is declared as a `bool`, so this cast shouldn't be required (it's not done in other components). There was a React error in this component, but that was related to: https://github.com/dhis2/ui-core/pull/246/files#diff-37738b57acc09fcc6cfc8af22d0933b5L124

`focus` isn't actually an attribute on an `input` element, so React threw an error for it. We use `componentDidMount` to set focus anyway so it isn't needed.